### PR TITLE
:book: add readme files

### PR DESCRIPTION
**Pull request by roblen45**
**Changes:**

1. :book: readme files added

PR: #8 

---

See the [examples](#example) for inspiration.

## How can others test the changes?

> Describe how others can test your changes (you can remove this section for typo fixes etc.)

## PR-Checklist

Please check the boxes in this list after submitting your PR:

- [x] You can merge this PR **right now** (if not, please prefix the title with "WIP: ")
- [x] I've tested **all** changes included in this PR.
- [x] I've also reviewed this PR myself before submitting (e.g. for scrambled letters, typos, etc.).
- [x] I've rebased the `develop` branch with my branch before finishing this PR.
- [x] I've **summarized all changes** in a list above.
- [x] I've mentioned all **PRs, which relate to this one**.
- [x] I've prefixed my Pull Request title is according to [gitmoji guide](https://gitmoji.carloscuesta.me/).

## Example

<details>
<summary>
:warning: Before merging please click here to expand and follow the guide.
</summary>
<br>

Please use `:twisted_rightwards_arrows:` at the beginning of your merge commit title.

Example 1:

<pre>
<code>
:twisted_rightwards_arrows: :bug: Fix Wrong Text Decoration at ...
</code>
</pre>

To get your commit message, just copy the first part of this pull request.

Example 2:
<pre>
<code>
**Changes:**

- Fixes Wrong Text Decoration at ...
- Fixes some typos
- ...

**Issues:**

Closes #NumberOfFixedIssue

PR: #NumberOfThisPR
</code>
</pre>
</details>

## Emojis

<details>
<summary>
Expand for a list of most used Emojis.
</summary>
<br>

Please prefix your commit messages with an Emoji.

Ref: https://gitmoji.carloscuesta.me/

| Description              | Glyphe               | Emoji  |
|--------------------------|----------------------|--------|
| Bugfix                   | `:bug:`              | 🐛     |
| Fixing Security Issues   | `:lock:`             | 🔒     |
| Configuration releated   | `:wrench:`           | 🔧     |
| Cosmetic                 | `:lipstick:`         | 💄     |
| Dependencies Downgrade   | `:arrow_down:`       | ⬇️     |
| Dependencies Upgrade     | `:arrow_up:`         | ⬆️     |
| Formatting               | `:art:`              | 🎨     |
| Improving Performance    | `:zap:`              | ⚡️      |
| Initial commit           | `:tada:`             | 🎉     |
| Linter                   | `:rotating_light:`   | 🚨     |
| Miscellaneous            | `:package:`          | 📦     |
| New Feature              | `:sparkles:`         | ✨     |
| Refactoring Code         | `:recycle:`          | ♻️      |
| Releasing / Version tags | `:bookmark:`         | 🔖     |
| Removing Stuff           | `:fire:`             | 🔥     |
| Tests                    | `:white_check_mark:` | ✅     |
| Work In Progress (WIP)   | `:construction:`     | 🚧     |

</details>

### CODE REVIEW

File: dotnet/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+This is an ExternalTaskApiClient written in C# for .NET Core.
+
+### Requirements
+
+- .NET Core 2.1
+
+### Setup/Installation
+
+Install all Dependencies
```
Comment (ElRaptorus) [inline dotnet/README.md]: `.` missing

File: dotnet/README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+dotnet restore
+```
+
+## Install from myget.org
+
+Install package with nuget:
+
+```bash
+dotnet add package ProcessEngine.ExternalTaskAPI.Client
+```
```
Comment (ElRaptorus) [inline dotnet/README.md]: Maybe add info about building and starting the app, using`dotnet build` and `dotnet run`, to make the setup section complete.

File: typescript/README.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# ExternalTaskApiClient for node.js
```
Comment (ElRaptorus) [inline typescript/README.md]: `Node.js` plz

File: typescript/README.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+This is an ExternalTaskApiClient written in typescript for node.js.
```
Comment (ElRaptorus) [inline typescript/README.md]: `TypeScript` and `Node.js` plz

File: typescript/README.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+
+### Requirements
+
+- npm >= 9.0
```
Comment (ElRaptorus) [inline typescript/README.md]: Also add the required Node.js Version here (should be >= 8.9.4).

File: typescript/README.md [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+
+### Setup/Installation
+
+Install all Dependencies
```
Comment (ElRaptorus) [inline typescript/README.md]: missing `.`

File: typescript/README.md [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+npm install
+```
+
+## Install from npm.org
+
+Install package with npm.org:
+
+```bash
+npm i @process-engine/external_task_api_client
+```
```
Comment (ElRaptorus) [inline typescript/README.md]: Maybe add info about building and starting the app, using`npm run build` and `npm start`, to make the setup section complete.

File: README.md [hunk 8]
```diff
@@ -1 +1,12 @@
-# external_task_api_client
+# ExternalTaskAPI Clients
+
+In this repository, we keep all client implementations for the
+ExternalTaskAPI we have done so far; you can currently find the
+following clients:
```
Comment (ElRaptorus) [inline README.md]: Better:

```md
This repository collects all implementations for the ExernalTaskApiClient.

The following implementations are currently available:
```

File: typescript/README.md (hunk 5 above)
Comment (cmg-dev) [reply typescript/README.md]: done.

File: dotnet/README.md (hunk 2 above)
Comment (ElRaptorus) [reply dotnet/README.md]: fair enough.

File: dotnet/README.md [hunk 9]
```diff
@@ -0,0 +1,23 @@
+# ExternalTaskApiClient for .NET Core
+
+This is an ExternalTaskApiClient written in C# for .NET Core.
+
+## Requirements
+
+- .NET Core 2.1
```
[the remark this answers is not in the document]
Comment (cmg-dev) [reply dotnet/README.md]: Ich glaube netstandard ist hier gemeint; macht nix, ist trotzdem falsch ;)

https://github.com/process-engine/external_task_api_client/blob/develop/dotnet/src/ProcessEngine.ExternalTaskAPI.Client/ProcessEngine.ExternalTaskAPI.Client.csproj#L4

offensichtlich zielen wir auf netstandard 2.0.

[the remark this answers is not in the document]
Comment (cmg-dev) [reply dotnet/README.md]: Ist korrigiert.

**Comment by cmg-dev** [comment]
@moellenbeck Dein Approve noch, dann merge ich das.

**Comment by Paulomart** [comment]
@roblen45 In dieser Branch liegt kein `Jenkinsfile-python`, daher _kann_ der Jenkins das nicht bauen. Dieser Branch muss rebased werden.

**Pull request merged**